queries/snowflake_queries/snowflake_brute_force_username_query.yml

```
@@ -19,7 +19,7 @@ Query: |
     DATEDIFF(HOUR, event_timestamp, CURRENT_TIMESTAMP) < 24
     AND event_type = 'LOGIN'
     AND error_code IS NOT NULL
-    AND error_code IS NOT 394304 # Ignore JWT Fingerprint Mismatch
+    AND error_code != 394304 # Ignore JWT Fingerprint Mismatch
```

Comment character again
```suggestion
    AND error_code != 394304 -- Ignore JWT Fingerprint Mismatch
```